TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Repository/NodeDataRepository.php

```diff
@@ -1424,9 +1424,10 @@ public function findByPathWithoutReduce($path, Workspace $workspace, $includeRem
      * )
      *
      * @param array $relationMap
+     * @param string $pathPrefix
      * @return array
      */
-    public function findNodesByRelatedEntities($relationMap)
+    public function findNodesByRelatedEntities($relationMap, $pathPrefix = '')
```

Comment:
In a better world we would not add an optional parameter but create a new method `findNodesByPathAndRelatedEntities(…)`